pkg/mocks/simplelogger.go

```diff
@@ -0,0 +1,69 @@
+package mocks
```

Comment:
What's the use of this logger?

Reply:
The `ReaperRequest{}` needs a logger and I couldn't find a suitable one. I'd appreciate any alternative suggestions you can provide so we can eliminate this mock.

Reply:
Removed this in latest revision.

Reply:
Mock logger removed.